docs/requirements.txt

```diff
@@ -1,7 +1,6 @@
 numpy>=1.17.0
 m2r2>=0.2.7
 sphinx-automodapi
-sphinx_rtd_theme
-sphinx==5.3.0
-sphinx_rtd_theme==1.1.1
-readthedocs-sphinx-search==0.1.1
\ No newline at end of file
+sphinx>=5.3.0 # cannot be updated due to m2r2 no longer being maintained -> switch to MyST?
```

Comment:
I know neither of the two, happy to switch to a maintained theme so we can keep sphinx up to date too yes!